samples/ingestion/ingestion-client/FetchTranscription/Language/AnalyzeConversationsProvider.cs

```diff
@@ -219,6 +220,43 @@ public async Task<bool> ConversationalRequestsCompleted(IEnumerable<AudioFileInf
             return errors;
         }
 
+        private static IEnumerable<ErrorEntity> GetAllErrorsFromResults((IEnumerable<AnalyzeConversationPiiResults> piiResults, IEnumerable<AnalyzeConversationSummarizationResults> summarizationResults, IEnumerable<string> errors)[] results)
+        {
+            var resultErrors = new List<ErrorEntity>();
+
+            if (results == null)
+            {
+                return resultErrors;
+            }
+
+            foreach (var result in results)
+            {
+                if (result.piiResults != null)
+                {
+                    foreach (var piiResult in result.piiResults)
+                    {
+                        if (piiResult.Errors != null)
+                        {
+                            resultErrors.AddRange(piiResult.Errors);
+                        }
+                    }
+                }
+
+                if (result.summarizationResults != null)
+                {
+                    foreach (var summarizationResult in result.summarizationResults)
+                    {
+                        if (summarizationResult.Errors != null)
```

Comment:
add a null check here for the summarizationResult to be sure.

Reply:
updated to the code you shared below